content/SCALE/SCALEUIReference/Storage/Datasets/SnapshotsScreens.md

```diff
@@ -9,36 +9,36 @@ tags:
 
 {{< toc >}}
 
-Use the **Snapshots** screen to manage existing snapshots or to add new snapshots. To access the **Snapshots** screen, from the **Storage** screen, click the **Snapshots** button in the top right of the screen and select **Snapshots**.
+Use the **Snapshots** screen to manage existing snapshots or to add new snapshots. To access the **Snapshots** screen, go to **Data Protection** > **Periodic Snapshot Tasks** and click the **Snapshots** button in the lower right corner of the widget.
```

Comment:
There are two ways to access the Snapshots screen, one from Data Protection > Periodic Snapshot Tasks using the Snapshots button the other from the Datasets screen using the Manage Snapshots on the Data Protection widget on that screen. Create a snippet for this if you want to create and maintain the content in one place and then insert the snippet where ever you talk about access the Snapshots screen and list of snapshots.

Also, this is the old Angelfish method so for some reason you are editing that content and not the restored Snapshots Screen. Not sure how that happened after Tim restored the article unless you didn't pull the update down to your github desktop.